### PR TITLE
fix: initial schemas

### PR DESCRIPTION
**Pull request by soedirgo**
Forgot to update w/ new migrations when upgrading the image versions in https://github.com/supabase/cli/commit/2990b5729f73f687fc352fe5baf6d86f973028fa

### CODE REVIEW

File: internal/utils/templates/initial_schemas/14.sql [hunk 1]
```diff
@@ -878,28 +962,31 @@ ALTER FUNCTION realtime.check_equality_op(op realtime.equality_op, type_ regtype
...
+                            val_2:=f.value
+                        )::int
+                    ) = count(1),
+                    true
+                )
+            from
+                unnest(filters) f
+                join unnest(columns) col
+                    on f.column_name = col.name;
+        $$;
```
Comment (sweatybridge) [inline internal/utils/templates/initial_schemas/14.sql:989]: we should probably have consistent indentations to diffs easier to read

**Review by sweatybridge — APPROVED** [review]
lgtm. tested 14 locally on macos x86

File: internal/utils/templates/initial_schemas/README.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ services:
 
   storage:
     container_name: supabase-storage
-    image: supabase/storage-api:v0.15.0
+    image: supabase/storage-api:v0.16.6
```
Comment (sweatybridge) [inline internal/utils/templates/initial_schemas/README.md:63]:
```suggestion
    image: supabase/storage-api:v0.18.7
```

Comment (soedirgo) [reply internal/utils/templates/initial_schemas/README.md:63]: 0.16.6 has the latest migrations. 0.18.7 is giving segfaults:
```
2022-08-23T05:41:56: PM2 log: Launching in no daemon mode
2022-08-23T05:41:56: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:56: PM2 log: App [server:0] online
2022-08-23T05:41:57: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:57: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:57: PM2 log: App [server:0] online
2022-08-23T05:41:58: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:58: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:58: PM2 log: App [server:0] online
2022-08-23T05:41:58: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:58: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:58: PM2 log: App [server:0] online
2022-08-23T05:41:59: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:59: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:59: PM2 log: App [server:0] online
2022-08-23T05:41:59: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:41:59: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:41:59: PM2 log: App [server:0] online
2022-08-23T05:42:00: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
2022-08-23T05:42:00: PM2 log: App [server:0] starting in -fork mode-
2022-08-23T05:42:00: PM2 log: App [server:0] online
2022-08-23T05:42:00: PM2 log: App [server:0] exited with code [0] via signal [SIGSEGV]
[...]
```

File: internal/utils/templates/initial_schemas/14.sql (hunk 1 above)
Comment (soedirgo) [reply internal/utils/templates/initial_schemas/14.sql:989]: probably needs to be fixed in the realtime/gotrue/storage migrations, schema diffs are whitespace-sensitive for function body (it's stored as plain string)

**Pull request merged**